www/locales/en.json

```diff
@@ -301,6 +301,8 @@
     "PUNJABI_TEEKA": "Punjabi Teeka",
     "TRANSLATION": "Translation",
     "TRANSLITERATION": "Transliteration",
+    "RESET FONT SIZES": "Reset Font Sizes",
```

Comment:
@yadvirkaur Can you please use underscores instead of spaces in the key. We have that standard followed throughout this file.

It would become: RESET_FONT_SIZE, and so on.

Reply:
Done. Thank you for noticing, it just skipped.